flutter-studio/src/io/flutter/profiler/Memory.java

```diff
@@ -111,354 +116,365 @@ public Object getValueAt(int rowIndex, int columnIndex) {
+        DefaultMutableTreeNode node = (DefaultMutableTreeNode)(_classesRoot.getChildAt(rowIndex));
+        return (Memory.ClassNode)(node.getUserObject());
+      }
     }
-  }
 
-  DefaultMutableTreeNode _classesRoot;
-  DefaultTreeModel _myClassesTreeModel;
+    DefaultMutableTreeNode _classesRoot;
+    DefaultTreeModel _myClassesTreeModel;
 
-  DefaultMutableTreeNode _myInstancesRoot;
-  DefaultTreeModel _myInstancesTreeModel;
+    DefaultMutableTreeNode _myInstancesRoot;
+    DefaultTreeModel _myInstancesTreeModel;
 
-  // All Classes (unfiltered) associated with the ClassesTable.
-  List<AllClassesInformation> _allClassesUnfiltered;
+    // All Classes (unfiltered) associated with the ClassesTable.
+    List<AllClassesInformation> _allClassesUnfiltered;
 
-  // Used to signal when all ClassRefs have been interrogated.
-  private int runningClassInfo;
-  private int filteredClasses;        // Total filtered classes displayed in snapshot
+    // Used to signal when all ClassRefs have been interrogated.
+    private int runningClassInfo;
+    private int filteredClasses;        // Total filtered classes displayed in snapshot
 
-  public void resetFilteredClasses() {
-    filteredClasses = 0;
-  }
+    public void resetFilteredClasses() {
+      filteredClasses = 0;
+    }
 
-  public int getFilteredClassesCount() {
-    return filteredClasses;
-  }
+    public int getFilteredClassesCount() {
+      return filteredClasses;
+    }
 
-  protected class InstanceNode {
-    static final String ROOT_POSTFIX = " Instances";
+    protected class InstanceNode {
+      static final String ROOT_POSTFIX = " Instances";
 
-    private String _objectRef;
+      private String _objectRef;
 
-    InstanceNode(String objectRef) {
-      this._objectRef = objectRef;
-    }
+      InstanceNode(String objectRef) {
+        this._objectRef = objectRef;
+      }
 
-    String getObjectRef() {
-      return _objectRef;
-    }
+      String getObjectRef() {
+        return _objectRef;
+      }
 
-    void setObjectRef(String newRef) { _objectRef = newRef; }
+      void setObjectRef(String newRef) { _objectRef = newRef; }
 
-    DefaultTreeModel getInstancesModel() { return _myInstancesTreeModel; }
+      DefaultTreeModel getInstancesModel() { return _myInstancesTreeModel; }
 
-    void setRootSize(int size) {
-      if (_objectRef.endsWith(ROOT_POSTFIX)) {
-        _objectRef = size + ROOT_POSTFIX;
+      void setRootSize(int size) {
+        if (_objectRef.endsWith(ROOT_POSTFIX)) {
+          _objectRef = size + ROOT_POSTFIX;
+        }
       }
-    }
 
-    @Override
-    public String toString() {
-      return getObjectRef();
+      @Override
+      public String toString() {
+        return getObjectRef();
+      }
     }
-  }
 
-  protected class ClassNode {
-    private ClassRef _ref;
-    private int _byteSize;
-    private int _instanceCount;
-    private List<String> _instanceIds;
+    protected class ClassNode {
+      private ClassRef _ref;
```

Comment:
should have caught this before but while you are here strip the Dart style leading _ from private member names.

Reply:
Done.